aquarius/app/assets.py

```diff
@@ -305,15 +312,38 @@ def validate_remote():
     responses:
       200:
         description: successfully request.
-        example:
-          application/json: true
       400:
         description: Invalid DDO format
       500:
         description: Error
     """
+    if request.content_type != "application/octet-stream":
+        return (
+            jsonify(
+                [
+                    {
+                        "message": "Invalid request content type: should be application/octet-stream"
+                    }
+                ]
+            ),
+            400,
+        )
+
+    pk = os.environ.get("PRIVATE_KEY", None)
+    values = {"hash": "", "publicKey": "", "r": "", "s": "", "v": ""}
+    if pk:
+        raw = request.get_data()
```

Comment:
my opinion is to move the contents of this if to a different function and only call it later, when we need to return it (line 362 currently, when doing "if valid")